### DISCUSSION TITLE
MOC Licence Key for DA-100

### DISCUSSION THREAD

**Discussion by laxmipriyakasi25** [github/india#805] [Students]
Hello!

I've received MOC Licence key for DA-100 via mail from [@ey-mentor1] on 21st Feb. But I wasn't able to redeem it in skillpipe. For Context, I was able to redeem around 3-4 with the licence keys previously.

When I try to redeem it, it is showing this...

![image](https://user-images.githubusercontent.com/85577800/158016019-c52c9486-27cc-4360-a707-da5360b77402.png)

I did reply back to the mail regarding this but got no response.
I am planning on using services based DA-100 for my project so I would really apprepriate if i can get the access to this.

**Comment by ey-mentor1** [comment]
Hello,
Can you please share the issue to same mail ID from which you received the licence.
The licence distribution is handled by other team.
Please add all relevant details and login information in the mail.
Thank you
